tcms/testplans/models.py

```diff
@@ -94,15 +94,22 @@ def list(cls, query=None):
     def confirmed_case(self):
         return self.case.filter(case_status__name='CONFIRMED')
 
-    def add_case(self, case, sortkey=0):
+    def add_case(self, case, sortkey=None):
 
-        tcp, is_created = TestCasePlan.objects.get_or_create(
+        if sortkey is None:
+            lastcase = self.testcaseplan_set.order_by('-sortkey').first()
```

Comment:
Is there any difference (mainly in speed I guess) between:
`order_by('-sortkey').first()` and `order_by('sortkey').last()` ?

Reply:
Hah, I didn't even think of using `last()`, I assume it results in the same query, and might actually be a bit more readable indeed. However, I see you have merged the request (thank you) so I guess it's not worth changing it now.

Reply:
More readable perhaps yes, speedwise we need to measure. If you feel like digging into it you can always send another PR.